starter/js/atm.js

```diff
@@ -1,15 +1,47 @@
 //Begin with the document ready function
-
+$(document).ready(function () {
+  $('#depositChecking').on('click', depositChecking);
+  $('#withdrawChecking').on('click', withdrawChecking);
+  $('#depositSavings').on('click', depositSavings);
+  $('#withdrawSavings').on('click', withdrawSavings);
+  $('#checkingBalance').addClass('zero');
```

Comment:
Do you want to always have the class of zero applied once the DOM is loaded? What if this was linked to a real backend that had money in the account. You would only want to apply that class based on the amount inside.